src/admin/redirect-detail/redirect-detail.service.ts

```diff
@@ -0,0 +1,179 @@
+import { stringifyUrl } from 'query-string';
+
+import { CustomError } from '../../shared/helpers/custom-error';
+import { getEnv } from '../../shared/helpers/env';
+
+import { mapTechnicalPath } from './helpers/map-technical-path';
+import { ITEMS_PER_PAGE } from './redirect-detail.const';
+import { type RedirectDetail, type RedirectDetailFilters } from './redirect-detail.types';
+
+export class RedirectDetailService {
+	public static async fetchAllRedirectDetails(): Promise<{ [avo1Path: string]: string }> {
+		const url = `${getEnv('PROXY_URL')}/redirect-details/all`;
```

Comment:
Should we rename the backend endpoint: /url-redirects instead of /redirect-details?